assets/xml/objects/object_rs.xml

```diff
@@ -28,7 +28,7 @@
         <Texture Name="gBombShopkeeperSkinTex" OutName="bombshopkeeper_skin" Format="ci8" Width="8" Height="8" Offset="0x5218" />
```

Comment:
```suggestion
        <Texture Name="gBombShopkeeperSkinTex" OutName="bomb_shopkeeper_skin" Format="ci8" Width="8" Height="8" Offset="0x5218" />
```
etc., since it's 2 words.